collin-newman/property-finder
# [Feature] Add routing

### Whats the problem being solved?
The app does not have any form of routing.

### How is the problem being solved?
Routing was added using expo-router. Expo router uses file-based routing, any files in the 'app' directory will be added automatically. Any file in the app directory becomes a 'Screen', screens must have a react component as the default export.

### How can I test it?
1. Fill out the form on the home page. Click submit.
2. Verify that you are routed to the property details page.
3. Verify the navigation buttons work correctly on each platform.

### Related docs/links:

[Expo Docs](https://docs.expo.dev/guides/routing-and-navigation/)
